data/sql_incremental_aggregates/prepare_schedule_a_aggregate_employer.sql

```diff
@@ -14,60 +14,17 @@ and is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, c
 group by cmte_id, cycle, employer
 ;
 
-alter table ofec_sched_a_aggregate_employer_tmp add column idx serial primary key;
+alter table ofec_sched_a_aggregate_employer_tmp add column idx serial;
+alter table ofec_sched_a_aggregate_employer_tmp add constraint uq_cmte_id_cycle_employer unique (cmte_id, cycle, employer);
```

Comment:
If we do need to keep a separate unique column, I take it this would have to be taken out?

Reply:
Even if we need to keep a separate unique column, this can still remain as you can have multiple unique constraints.